float64 trackObjArea
float64 trackObjX
float64 trackObjY
